install.md

```diff
@@ -7,12 +7,45 @@ RHEL/CentOS ≥ 8 or Fedora you can install it using:
 $ sudo dnf install skopeo
 ```
 
+Newer skopeo releases may be available on the repositories provided by the
+Kubic project. Beware, these have not been blessed by the official Red Hat QA
```

Comment:
Since this is not Red Hat, then we should not mention them. Nothing in Centos goes through Red Hat QA.

Reply:
ah yes, i'll check what I have in podman and update this one.

Reply:
so the podman.io repo mentions Red Hat QA as well. So it needs to be changed there as well in that case.